cassdegrees/templates/createprogram.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% block page-title %}Create Program Template{% endblock %}
+
+{% block subtitle %}Create Program Template{% endblock %}
+
+{% block content %}
+    {% if msg != None %}
+        <p class="{% if is_error %}msg-error{% else %}msg-success{% endif %}">{{ msg }}</p>
+    {% endif %}
+
+
+    <form class="anuform" action="" method="post">
+        {% csrf_token %}
+
+        <fieldset>
+            <legend>Configure Program Details</legend>
+
+            <p>
+                <label for="code">Program Code:</label>
+                <input class="text" id="code" type="text" name="code" aria-required="true" placeholder="e.g. BARTS" required>
+            </p>
+
+            <p>
+                <label for="name">Program Name:</label>
+                <input class="text" id="name" type="text" name="name" aria-required="true" placeholder="e.g. Bachelor of Arts" required>
+            </p>
+
+            <p>
+                <label for="year">Program Year:</label>
+                <input class="text" id="year" type="number" min="2000" max="2100" value="2019" name="year" aria-required="true" required>
```

Comment:
```suggestion
                <input class="text" id="year" type="number" min="2000" max="2100" value="{{ year|default:"2019" }}" name="year" aria-required="true" required>
```

Makes the year value autofill once the page is sent